src/openforms/authentication/signals.py

```diff
@@ -59,19 +63,43 @@ def set_auth_attribute_on_session(
         )
         raise PermissionDenied(_("Demo plugins require an active admin session."))
 
+    # TODO check this log is accurate
     logger.debug(
         "Persisting form auth to submission %s. Plugin: '%s' (setting attribute '%s')",
         instance.uuid,
         plugin,
         attribute,
     )
```

Comment:
I'd include an additional info item whether a registrator initiated this or not (basically: `registrator_subject is not None` - or maybe better -> rely on `request.user` to provide this info.